### ISSUE TITLE
Bug: role="button" and translation support with aria-label

### ISSUE THREAD

**Issue by briannhinton** [nuejs/create-nue#45]
The role="button" attribute on the links changes the semantic meaning for screen reader users. Additionally using aria-label should be the last choice with elements as they are not supported by translators in the browser.

**Comment by tipiirai** [comment]
Gotcha. How would you implement a button-looking link element with that does not require JavaScript?
